app/services/login/user_verifier.rb

```diff
@@ -137,6 +138,17 @@ def attempt_secondary_idme_identifier
       Rails.logger.info("[Login::UserVerifier] Attempting alternate type=#{type}  identifier=#{identifier}")
     end
 
+    def linked_user_verification_is_locked?
+      return false unless existing_user_account &&
```

Comment:
since the log has to be changed, this can be simplified, now you can just do the conditional and return that as the status whether locked should be true or false:

`existing_user_account.user_verifications.send(login_type).where(locked: true).any?`